addons/sale_timesheet/models/project_update.py

```diff
@@ -21,12 +21,15 @@ def _get_template_values(self, project):
             'show_activities': template_values['show_activities'] or show_sold or bool(profitability),
             'services': services,
             'profitability': profitability,
+            'format_value': lambda value, unit: str(round(value, 2)) if unit != 'Hours'\
```

Comment:
`unit` could be translated, I think it will be better if we can use the `is_unit` variable instead. 🤞🏽

Reply:
is_hour